demo/src/demo-registry.ts

```diff
@@ -0,0 +1,220 @@
+import * as Cord from '@cord.network/sdk'
+import { UUID } from '@cord.network/utils'
+
```

Comment:
While the file says its 'demo-registry', i didn't see any Registry module calls. Otherwise the PR seems good.